test/tracerEq/test_prescribed_velocity.py

```diff
@@ -0,0 +1,122 @@
+"""
+Simple 2D tracer advection problem in a doubly periodic domain with prescribed fluid velocities.
```

Comment:
This test case advects a Gaussian bump in a periodic channel with a constant or sinusoidal (in time) velocity. In the end, L2 error is computed. Note that `test_h-advection_mes_2d.py` already advects a (1D) Gaussian and computes the L2 error and convergence rate. As in principle this test does not add to the test coverage, it might be a better idea to extend `test_h-advection_mes_2d.py` instead: add periodic mesh, 2D Gaussian source and optionally sinusoidal velocity. Our test suite is already quite heavy so we shouldn't add unnecessary tests.